### PR TITLE
Add more boilerplate comments

### PR DESCRIPTION
**Pull request by dpikt**
Should help people get running faster with new subsections, and lower the frustration of writing the same imports again and again.

This PR also updates the default path to the one used in the client template.

### CODE REVIEW

**Comment by dpikt** [comment]
PS if this is approved I will also edit the `ReactReduxComponent` snippet to match.

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You can run the `generate-subsection` command using `yarn`:
 > yarn generate-subsection {section-name} {path}
 ```
 
-The `path` option should be the path to the directory where you'd like the subsection to be generated (ex. `./src/js/main/`).
+The `path` option should be the path to the directory where you'd like the subsection to be generated (optional, default=`./src/js/main/`).
```
Comment (chawes13) [inline README.md]: 🙌 long overdue

File: template/Routes.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ function Routes ({ match: { path } }) {
     <Layout>
       <Switch>
         <Route exact path={ path + '/' } component={ Views.%SubSection% } />
+        {/* <Route path={ path + '/:id' } component={ Views.%SubSection%Details } /> */}
```
Comment (chawes13) [inline template/Routes.js]: Do you want to align more with RESTful nomenclature and call it `%SubSection%Show`?

Comment (chawes13) [reply template/Routes.js]: What's stopping you from actually creating this view?

Comment (dpikt) [reply template/Routes.js]: Calling it `show` is a good idea. In terms of generating it, depends whether you think we use that view >50% of the time. Probably, right?

**Comment by dpikt** [comment]
Yeah, this is definitely aimed at newer devs. However, some of the commented stuff will be helpful regardless (the imports and HOCs) and probably faster to ctrl-f+r `thing` (or `resource`) than type it up from scratch. I also think having references to the libraries in the imports will enable people to look up the docs if they want.

One idea I was playing around with was adding a "terse" mode that removes some of the extra comments for more experienced users- would this address your concerns?

**Comment by dpikt** [comment]
Another idea would be to name that `resource` after the subsection itself. It might verge on being too opinionated, but that along with including the `show` view could be pretty powerful.

**Comment by chawes13** [comment]
I like the idea of naming it after the subsection. I also really like the idea of a terse mode!

**Comment by dpikt** [comment]
`--power-user` 😀

**Comment by chawes13** [comment]
That flag can safely be included in a v2 of this enhancement though. No need to block this PR

**Comment by chawes13** [comment]
This is really cool

**Pull request merged**